style.scss

```diff
@@ -0,0 +1,321 @@
+$mainColor: #ffe8da;
+$highlightColor: #e0b1af;
+$textFont:'Poppins', sans-serif;
+$headingFont: 'Nunito', sans-serif;
+$itemPadding:20px;
+$textColor:#2e2e2e;
+$headingColor:rgb(30, 53, 117);
+$textspacing:0.5px;
+$zeroMargin:0;
+$textAlign:justify;
+
+$sectionBorder:1px solid #808080;
+
+@mixin sections {
+    padding:75px 0;
+    border-bottom: $sectionBorder;
+}
+
+@mixin  subSection($display, $columnTemplate, $gap) {
+    display: $display;
+    grid-template-columns:$columnTemplate;
+    gap:$gap;
+    color: $textColor;
+    text-align: $textAlign;
+    letter-spacing: $textspacing;
+}
+
+@mixin headingSection {
+    font-family: $headingFont;
+    margin: $zeroMargin;
+    margin-bottom:25px;
+    color: $headingColor;
+}
+
+@mixin formCells {
+    padding: $itemPadding;
+    margin:10px 10px 10px 0;
+    font-family: $headingFont;
+}
+
+@mixin btn {
+    padding: 20px 40px;
+    font-family: $textFont;
+    font-weight: bold;
+}
+
+@mixin contacts {
+    font-family: $textFont;
+    margin: 16px;
+    justify-self: start;
+}
+
+@mixin contactIcon {
+    max-width: 15%;
+    justify-self: end;
+}
+
+body{
+    margin:$zeroMargin;
+    background-color: $mainColor;
+}
+
+.header{
+    display: grid;
+    grid-template-columns: 100px auto 1fr;
+    margin: 0 50px;
+    border-bottom: $sectionBorder;
+    background-color: #ffe8d5;
+    z-index: 10;
+        &--sticky{
+            position: sticky;
+            top: 0;
+        }
+        &__photo-of-me{
+            width: 100%;
+            border-radius: 50%;
+            place-self: center center;
+        }
+        &__name{
+            font-family: $headingFont;
+            font-size: 30px;
```

Comment:
Font-size má taky lepší jednotky než jsou pixely. Pěkný článek si o tom můžeš přečíst [tady](https://bitsofco.de/css-font-sizing/).

Reply:
Z těch relativních jednotek jsem trochu jelen :O Ještě si to musím pořádne nastudovat, nicméně děkuji za odkaz na články.